src/MSONNamedTypeParser.h

```diff
@@ -15,6 +15,9 @@ using namespace scpl;
 
 namespace snowcrash {
 
+    /** MSON reserved characters matching regex */
+    const char* const MSONReservedCharsRegex = "[]:\()<>\{}[_*+`-]?";
```

Comment:
Actually, the backticks will be stripped, so we won't be having any issues here. I thought there might be issues with nested escaped characters, but that case will also not be a problem here. So, the regex is correct.